GLMakie/src/gl_backend.jl

```diff
@@ -17,7 +17,6 @@ using .GLAbstraction
 const atlas_texture_cache = Dict{Any, Tuple{Texture{Float16, 2}, Function}}()
 
 function get_texture!(atlas)
-    Makie.set_glyph_resolution!(Makie.High)
```

Comment:
https://github.com/JuliaPlots/Makie.jl/blob/8bab35403db92bb921738380d4c5ecff8df6ec37/WGLMakie/src/WGLMakie.jl#L49

Is this necessary? If it is we should probably have GLMakie set the glyph resolution on init, not here.

Reply:
> If it is we should probably have GLMakie set the glyph resolution on init, not here.

That's already the case.

Reply:
Uh, I think I added this, to make sure Makie gets the correct texture atlas - sadly this is a Makie global variable... E.g., when using GLMakie, then WGLMakie and then switching back to GLMakie you'll get pretty weird artifacts

Reply:
Ok, nvm, this works on an existing texture atlas...
`activate!` is the right place: https://github.com/JuliaPlots/Makie.jl/blob/master/GLMakie/src/GLMakie.jl#L42

Reply:
I used to get artifacts when doing `empty!(Makie.global_texture_atlas)` and removing the cached files. But I'm not getting them anymore. I don't think I got when switching between WGLMakie and GLMakie either.